docs/data-management-on-carbondata.md

```diff
@@ -284,6 +286,20 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
 	 ALTER TABLE employee SET TBLPROPERTIES (‘CACHE_LEVEL’=’Blocklet’)
 	 ```
 	 
+	 - **Support Flat folder**
```

Comment:
change to : 

**Support Flat folder same as Hive/Parquet**

Reply:
ok